### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by wlteo**
Wrote some documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,46 @@
...
+
+**Sabrina**, **Mark**, **Evan**
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details
+
+## Acknowledgments
+
+* Hat tip to anyone who's code was used
```
Comment (sabrina-s) [inline README.md]: *whose 🤓

**Review by sabrina-s — CHANGES_REQUESTED** [review]
Just one typo, the rest is good to go! 😸

File: README.md (hunk 1 above)
Comment (wlteo) [reply README.md]: whoops

**Review by sabrina-s — APPROVED** [review]
LGTM! 🚀

**Pull request merged**